HULKs/hulk
# Add and switch to ONNX

## Introduced Changes

Introduced ONNX models. Used basic `tf2onnx` utility, there are other ways too ;).

I would still keep the hdf5 models for now, at least until we figure out a good way to version & trace our model development!

Fixes #

CompiledNN has ONNX :P we want to use that!
In addition, the pure-rust tract-onnx inference backend can directly use these ;)

## Ideas for Next Iterations (Not This PR)

## How to Test

We should run a full round of test and see.
I'll run the micro-benchmarks I have and see if there is a performance difference.
